Sources/Main/CLIUtility.swift

```diff
@@ -29,9 +29,9 @@ public class CLIUtility {
 
     public func getDefaultDirectory() -> String? {
         if (!FileManager.default.fileExists(atPath: defaultDirectory)) {
-           return defaultDirectory 
+           return nil 
```

Comment:
Even with these changes you're still making the assumption that the directory they provide is formatted correctly, and exists. I would like to see you checking the user input directory as well!